Misc/NEWS.d/next/Core and Builtins/2022-01-17-07-34-16.bpo-46407.00aN9w.rst

```diff
@@ -0,0 +1 @@
+Optimized power-of-2 operations in ``Objects/longobject.c`` as part of bpo-46407. Patch by Jeremiah Vivian.
```

Comment:
```suggestion
Optimized power-of-2 operations in ``Objects/longobject.c``. Patch by Jeremiah Vivian.
```
This is redundant because it's already in the fiilename, and blurb automatically adds it to the changelog.

Reply:
Done.